modules/escalation/main.tf

```diff
@@ -5,17 +5,17 @@ resource "opsgenie_escalation" "this" {
   description   = try(var.escalation.description, var.escalation.name)
   owner_team_id = try(var.escalation.owner_team_id, null)
 
-  rules {
-    condition   = try(var.escalation.rule.condition, "if-not-acked")
-    notify_type = try(var.escalation.rule.notify_type, "default")
-    delay       = try(var.escalation.rule.delay, 0)
+  dynamic "rules" {
+    for_each = var.escalation.rules
 
-    dynamic "recipient" {
-      for_each = try(var.escalation.rule.recipients, [])
+    content {
+      condition   = try(rules.value.condition, "if-not-acked")
+      notify_type = try(rules.value.notify_type, "default")
+      delay       = rules.value.delay
```

Comment:
```suggestion
      delay       = try(rules.value.delay, 0)
```

Reply:
It's required indeed.

Reply:
I added this change with [d3fdc3bc4b8f74997c99f44e18901982e7007f96](https://github.com/cloudposse/terraform-opsgenie-incident-management/pull/58/commits/d3fdc3bc4b8f74997c99f44e18901982e7007f96) because it is required - by adding try if its not found it uses 0 - essentially defaulting the value.

Reply:
ok